{"uuid":"cd47a263-f9bc-4d33-871d-caa638598435","historyId":"73c14d8691c8eae35f5d6fc69fb407af","fullName":"testCases.LoginPopUpTest.loginPopUpLogInNegativeTest","labels":[{"name":"package","value":"testCases.LoginPopUpTest"},{"name":"testClass","value":"testCases.LoginPopUpTest"},{"name":"testMethod","value":"loginPopUpLogInNegativeTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"craftBetAutomation"},{"name":"subSuite","value":"testCases.LoginPopUpTest"},{"name":"host","value":"DESKTOP-PO3T4V3"},{"name":"thread","value":"12568@DESKTOP-PO3T4V3.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"severity","value":"blocker"}],"links":[],"name":"Validate on Log_in Pop Up Log_In functionality with invalid data","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Cannot invoke \"org.apache.log4j.Logger.info(Object)\" because \"testCases.BaseTest.logger\" is null","trace":"java.lang.NullPointerException: Cannot invoke \"org.apache.log4j.Logger.info(Object)\" because \"testCases.BaseTest.logger\" is null\r\n\tat testCases.BaseTest.StartSuite(BaseTest.java:38)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:568)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)\r\n\tat org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:372)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:330)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1176)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1099)\r\n\tat org.testng.TestNG.run(TestNG.java:1067)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\r\n"},"stage":"finished","description":"Validate on Log_in Pop Up Log_In functionality with invalid data","steps":[],"attachments":[],"parameters":[{"name":"arg1","value":"N3 HU3"},{"name":"arg0","value":"U1630370"}],"start":1657265919791,"stop":1657265919792}